test/cli-test.js

```diff
@@ -71,13 +72,17 @@ function captureStream(stream){
 describe('User Preferences:', () => {
   describe('When the user is asked about completing the prerequisites:', () => {
     let hook;
+    let previousLevel;
 
     beforeEach(() => {
       hook = captureStream(process.stdout);
+      previousLevel = winston.level;
```

Comment:
NOTE: This was a bit of scope creep, but I added it to make the tests run better. They would fail if you had the winston logger set to a level other than `info` since this test captures the console output.